src/simulation/xml_parser.cpp

```
@@ -210,32 +209,33 @@ void XMLParser::handleVehicle(const pugi::xml_node& vehicle_node,
 
   VehicleClassDef vehicle_definition = itr->second;
 
-  std::string pos_str = vehicle_node.child_value("init_pose");
+  std::string       pos_str = vehicle_node.child_value("init_pose");
   std::stringstream ss(pos_str);
-  std::string axis;
+  std::string       axis;
 
   glm::vec3 pos;
```

Clang (14.0.6)

[error] variable 'pos' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)